azext_edge/tests/edge/support/create_bundle_int/helpers.py

```diff
@@ -579,3 +600,39 @@ def _clean_up_folders(
         level_2 = walk_result.pop(path.join(BASE_ZIP_PATH, arc_namespace, "arcagents"))
         assert level_2["folders"] == [agent[0] for agent in ARC_AGENTS], f"Mismatch; folders: [{level_2['folders']}]"
         assert not level_2["files"]
+
+
+def _compare_support_bundle_names(
+    resource_type: str, bundle_names: str, pre_bundle_resources: dict, post_bundle_resources: dict
+):
+    """
+    Do the name comparison with some extra debug information.
+
+    Try to get labels for missing resources to help determine if labels are the reason.
+    """
+    try:
+        find_extra_or_missing_names(
+            resource_type=resource_type,
+            result_names=bundle_names,
+            pre_expected_names=pre_bundle_resources.keys(),
+            post_expected_names=post_bundle_resources.keys(),
+        )
+    except AssertionError as e:
+        error_text = str(e)
+        missing_error_text = error_text.rsplit("\n", maxsplit=1)[-1]
+        if "Missing " not in missing_error_text:
+            # cannot get labels from extra resources (only in support bundle)
```

Comment:
tbh in the extra resource case - I can try checking the prefixes and 
1. if they match up - assume the resource got created + deleted in the time frame -> warning
2. if they differ - error and print out the expected prefixes in addition to the names